microsoft-edge/devtools-guide-chromium/javascript/reference.md

```diff
@@ -366,6 +366,41 @@ If you find yourself running the same debug code in the **Console** tool over an
 See [Run snippets of JavaScript on any webpage](./snippets.md).
 
 
+<!-- ====================================================================== -->
+## Use `# sourceURL` to name evaluated and inline code in the Sources tool
```

Comment:
```suggestion
## Use `# sourceURL` to name evaluated and inline code in the Sources tool and throughout DevTools
```

Reply:
The virtual file name will be visible in the Console and Performance tool, at least. So, I'm changing this heading.